### ISSUE TITLE
Way to contribute: Hosting event

### ISSUE THREAD

**Issue by swarnava** [MozillaIndia/asknot#7]
As a Mozilla supporter i want to spread Mozilla missions in my local area/my college/my school by hosting small or big event. Lets create a wiki how to get start/whom should i contact/how to plan with good metrics and make it successful event

**Comment by sudheesh001** [comment]
I think this also needs a list of evangelists to whom the members can reach out.

**Comment by abhi12ravi** [comment]
@swarnava  Isn't this what [Vibe](https://github.com/MozillaIndia/vibe) plans to do?

**Comment by swarnava** [comment]
is it? not sure though.

**Comment by kumarrishav** [comment]
Nope, vibe only scrap out event of India from reps Event portal. Also there will be list of evangelist and their details.

**Comment by abhi12ravi** [comment]
@swarnava @kumarrishav  Okay, Vibe solves part of the issue. If I'm conducting an event, I can check up on the nearest available Mozillian, good at a particular topic/technology. The event organizer can then follow up with the Mozillian using the contact info provided there.

**Comment by sudheesh001** [comment]
@abhi12ravi That's the plan in the near future.
